### PR TITLE
Rename reject to 'request changes'

### LINKED ISSUE #5857: Suggestion: rename "Reject" to "Request changes" (or similar)
- Gitea version (or commit ref): d663930
(rest is irrelevant)

## Description

When reviewing a PR you can "Approve" or "Reject". The problem is that "Reject" sound too hard/rude and can hurt team relationship and confidence. Most of the time the PR is almost good but you are requesting some changes before merging, not rejecting the PR itself. GitHub, as an example, shows "Request changes" instead and I think it's a better name.

So, consider rename the "Reject" button and the "Someone rejected these changes" text to something like "Request changes" and "Someone requested changes".

### PR DESCRIPTION
**Pull request by adelowo**
Fixes https://github.com/go-gitea/gitea/issues/5857

GitHub also uses 'Request changes'

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Review by zeripath — APPROVED** [review]
Make LGTM work?

**Comment by 0x5c** [comment]
Isn't line [845](https://github.com/go-gitea/gitea/blob/1416fe5140f3a3c7dd4eb3259cf968b31cc43e3f/options/locale/locale_en-US.ini#L845) also to edit?

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1221,7 +1221,7 @@ diff.review.header = Submit review
 diff.review.placeholder = Review comment
 diff.review.comment = Comment
 diff.review.approve = Approve
-diff.review.reject = Reject
+diff.review.reject = Request changes
```
Comment (zeripath) [inline options/locale/locale_en-US.ini:1224]: You should edit line 845 as per @0x5c

**Review by zeripath — CHANGES_REQUESTED** [review]

**Review by zeripath — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
make LG-TM work

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -842,7 +842,7 @@ issues.review.self.rejection = You cannot request changes on your own pull reque
 issues.review.approve = "approved these changes %s"
 issues.review.comment = "reviewed %s"
 issues.review.content.empty = You need to leave a comment indicating the requested change(s).
-issues.review.reject = "rejected these changes %s"
+issues.review.reject = "requested these changes %s"
```
Comment (lafriks) [inline options/locale/locale_en-US.ini:845]: this is actually wrong and should be `requested changes %s`

Comment (zeripath) [reply options/locale/locale_en-US.ini:845]: Damn.

Can we change it in crowdin? Or do we need another PR?

Comment (lafriks) [reply options/locale/locale_en-US.ini:845]: need new PR

Comment (zeripath) [reply options/locale/locale_en-US.ini:845]: Just opened PR #5869

**Pull request merged**